docs/HowDoI_interface.md

```diff
@@ -0,0 +1,35 @@
+# How do I ?
```

Comment:
maybe this should be index.md?

I believe github renders index.md by default when you browse to a folder

Reply:
I don't see any such feature with [the official cheatsheet](https://guides.github.com/pdfs/markdown-cheatsheet-online.pdf)

I did find [this cli utility](https://github.com/ekalinin/github-markdown-toc#auto-insert-and-update-toc), but not anything related to GitHub. Could you please link the feature you referenced?

Reply:
apparently you need to call it readme.md, index doesn't work